pkg/controller/metrics/metrics.go

```diff
@@ -41,11 +42,26 @@ var (
 		Name: "hive_cluster_deployments_installed_total",
 		Help: "Total number of cluster deployments that are successfully installed.",
 	})
+	metricInstallJobsRunningTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "hive_install_jobs_running_total",
+		Help: "Total number of install jobs running in Hive.",
+	})
+	metricUninstallJobsRunningTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "hive_uninstall_jobs_running_total",
+		Help: "Total number of uninstall jobs running in Hive.",
+	})
+	// installJobLabel is the label used for counting the number of install jobs in Hive
+	installJobLabel = "hive.openshift.io/install"
+
+	// uninstallJobLabel is the label used for counting the number of uninstall jobs in Hive
+	uninstallJobLabel = "hive.openshift.io/uninstall"
```

Comment:
Lets define these labels in one place, remove them from this file and see comment below.